### ISSUE TITLE
Create and use a single-use link for a file accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3989]
### Descriptive summary

Descriptive summary
QA Testing for Hyrax
Hyrax Version: 3.0.0.pre.rc1
Hyrax Branch: master
Hyrax Revision: 48ed925
Nurax Version
ENV: production
SHA: 40b47b0d71af1b958e016ebe49212c1740683133
Branch: 2019-09-12-10-00
Updated: 12 Sep 2019 15:00:37

In testing for the Hyrax 3.0 release, the following points are actions that were discovered during a FireFox tab test and axe accessibility audit. Note, only certain issues are being reported at this time.

 **Elements must have sufficient color contrast**
![Screen Shot 2019-09-12 at 8 47 27 PM](https://user-images.githubusercontent.com/9621070/64836392-8e45d100-d59e-11e9-9b36-3b3738c490b5.png)

![Screen Shot 2019-09-12 at 8 47 49 PM](https://user-images.githubusercontent.com/9621070/64836404-9b62c000-d59e-11e9-8f9d-1d3389b94d4f.png)

![Screen Shot 2019-09-12 at 8 48 10 PM](https://user-images.githubusercontent.com/9621070/64836416-a9184580-d59e-11e9-8161-6b390b5239b9.png)

![Screen Shot 2019-09-12 at 8 48 33 PM](https://user-images.githubusercontent.com/9621070/64836430-b6cdcb00-d59e-11e9-93a4-fa162cdbb248.png)

**ARIA role must be appropriate for the element**
![Screen Shot 2019-09-12 at 8 49 10 PM](https://user-images.githubusercontent.com/9621070/64836442-cc42f500-d59e-11e9-96ae-288c2ee3c873.png)

**Banner landmark must not be contained in another landmark**
![Screen Shot 2019-09-12 at 8 49 46 PM](https://user-images.githubusercontent.com/9621070/64836458-dfee5b80-d59e-11e9-82e3-81b9c9ebc2dc.png)

**Document must not have more than one banner landmark**
![Screen Shot 2019-09-12 at 8 50 23 PM](https://user-images.githubusercontent.com/9621070/64836492-f5fc1c00-d59e-11e9-8011-3f31bfbd8ba3.png)

**Ensures landmarks are unique**
![Screen Shot 2019-09-12 at 8 50 49 PM](https://user-images.githubusercontent.com/9621070/64836507-0613fb80-d59f-11e9-974e-c43f75fccefe.png)

![Screen Shot 2019-09-12 at 8 51 42 PM](https://user-images.githubusercontent.com/9621070/64836525-2643ba80-d59f-11e9-9b06-60ac969fff4f.png)

**All page content must be contained by landmarks**
![Screen Shot 2019-09-12 at 8 52 11 PM](https://user-images.githubusercontent.com/9621070/64836540-36f43080-d59f-11e9-9573-eeaa6d77dcac.png)

### Steps to reproduce the axe accessibility audit

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/concern/file_sets/bk128b08j/edit?locale=en
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Video of Tab Test

Unable to click Create Single Use Link in Firefox. Chrome working fine. 

Video: https://cl.ly/bd0ffc886667/download/Screen%252520Recording%2525202019-09-12%252520at%25252008.45%252520PM.mov 

Link: https://nurax-dev.curationexperts.com/concern/parent/tq57nr202/file_sets/bk128b08j

**Comment by rjkati** [comment]
Tested using axe in Firefox on dev.nurax running Hyrax 5.0.0.rc2. Some issues have been addressed, but the following are still issues:

Elements must have sufficient color contrast
- Only the 4 contrast issues referring to Help, About, Home, Contact links still need to be addressed. Other contrast issues in ticket have been resolved.


Banner landmark must not be contained in another landmark


Additionally, a new issue `All page content must be contained by landmarks` seems to have been introduced, referring to `.blue-footer`, rather than `html`
![bluefooter](https://github.com/samvera/hyrax/assets/32644254/de1e04f1-0856-42d8-b32d-a5d63d8b384e)
